### ISSUE TITLE
npm install not working?

### ISSUE THREAD

**Issue by craig006** [ArsenyYankovsky/typeorm-aurora-data-api-driver#10]
Hi,

Glad to see typeorm has been release with the PR for this. 

I'm running into an issue that I'm not sure how to overcome. I installed this driver in my project and got a test Lambda up and running. The lambda produces this error:

```
2019-09-16T16:14:02.831Z 67f1cd3a-b3b5-4ca8-abe1-ad8fcfa50d52 { Error: Cannot find module '/var/task/node_modules/typeorm-aurora-data-api-driver'
at Function.Module._resolveFilename (module.js:547:15)
at Function.Module._load (module.js:474:25)
at Module.require (module.js:596:17)
at require (internal/module.js:11:18)
at Function.PlatformTools.load (/var/task/node_modules/typeorm/platform/PlatformTools.js:112:24)
at AuroraDataApiDriver.loadDependencies (/var/task/node_modules/typeorm/driver/aurora-data-api/AuroraDataApiDriver.js:664:60)
at new AuroraDataApiDriver (/var/task/node_modules/typeorm/driver/aurora-data-api/AuroraDataApiDriver.js:237:14)
at DriverFactory.create (/var/task/node_modules/typeorm/driver/DriverFactory.js:56:24)
at new Connection (/var/task/node_modules/typeorm/connection/Connection.js:52:59)
at ConnectionManager.create (/var/task/node_modules/typeorm/connection/ConnectionManager.js:56:26) code: 'MODULE_NOT_FOUND' }
```

After inspecting my node_modules directory I can see `typeorm-aurora-data-api-driver` in there but with no code inside? I would have expected a `bin` folder or `index.js` or something. Is this correct?

![image](https://user-images.githubusercontent.com/23329523/64975962-82b11d80-d8b0-11e9-9d09-6555dfacea41.png)

**Comment by ArsenyYankovsky** [comment]
My bad, should be fixed with 1.1.3
Will make the nightly build to run against latest released version

**Comment by spolom** [comment]
While researching TypoORM with AWS' Data API, I ran into the bellow error (simple serverless lambda w/ TypeScript). typeorm-aurora-data-api-driver is not present in the zipped lambda sources.

I am not sure whether it is related to this closed issue or a configuration error on my side.

`2021-01-18T16:41:16.410Z	e18a07b9-31ab-4496-a75b-43f7e30f991b	ERROR	Invoke Error 	
{
    "errorType": "Error",
    "errorMessage": "Cannot find module '/var/task/node_modules/typeorm-aurora-data-api-driver'\nRequire stack:\n- /var/task/node_modules/typeorm/platform/PlatformTools.js\n- /var/task/node_modules/typeorm/driver/sqlserver/SqlServerDriver.js\n- /var/task/node_modules/typeorm/migration/MigrationExecutor.js\n- /var/task/node_modules/typeorm/connection/Connection.js\n- /var/task/node_modules/typeorm/connection/ConnectionManager.js\n- /var/task/node_modules/typeorm/index.js\n- /var/task/src/functions/hello/handler.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
    "code": "MODULE_NOT_FOUND",
    "requireStack": [
        "/var/task/node_modules/typeorm/platform/PlatformTools.js",
        "/var/task/node_modules/typeorm/driver/sqlserver/SqlServerDriver.js",
        "/var/task/node_modules/typeorm/migration/MigrationExecutor.js",
        "/var/task/node_modules/typeorm/connection/Connection.js",
        "/var/task/node_modules/typeorm/connection/ConnectionManager.js",
        "/var/task/node_modules/typeorm/index.js",
        "/var/task/src/functions/hello/handler.js",
        "/var/runtime/UserFunction.js",
        "/var/runtime/index.js"
    ],
    "stack": [
        "Error: Cannot find module '/var/task/node_modules/typeorm-aurora-data-api-driver'",
        "Require stack:",
        "- /var/task/node_modules/typeorm/platform/PlatformTools.js",
        "- /var/task/node_modules/typeorm/driver/sqlserver/SqlServerDriver.js",
        "- /var/task/node_modules/typeorm/migration/MigrationExecutor.js",
        "- /var/task/node_modules/typeorm/connection/Connection.js",
        "- /var/task/node_modules/typeorm/connection/ConnectionManager.js",
        "- /var/task/node_modules/typeorm/index.js",
        "- /var/task/src/functions/hello/handler.js",
        "- /var/runtime/UserFunction.js",
        "- /var/runtime/index.js",
        "    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)",
        "    at Function.Module._load (internal/modules/cjs/loader.js:687:27)",
        "    at Module.require (internal/modules/cjs/loader.js:903:19)",
        "    at require (internal/modules/cjs/helpers.js:74:18)",
        "    at Function.PlatformTools.load (/var/task/src/platform/PlatformTools.ts:130:20)",
        "    at AuroraDataApiPostgresDriver.loadDependencies (/var/task/src/driver/aurora-data-api-pg/AuroraDataApiPostgresDriver.ts:104:17)",
        "    at new AuroraDataApiPostgresDriver (/var/task/src/driver/aurora-data-api-pg/AuroraDataApiPostgresDriver.ts:56:14)",
        "    at DriverFactory.create (/var/task/src/driver/DriverFactory.ts:65:24)",
        "    at new Connection (/var/task/src/connection/Connection.ts:128:43)",
        "    at ConnectionManager.create (/var/task/src/connection/ConnectionManager.ts:64:28)"
    ]
}
`

This is the webpack config:

`const path = require('path');
const slsw = require('serverless-webpack');
const nodeExternals = require('webpack-node-externals');
const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');

module.exports = {
  context: __dirname,
  mode: slsw.lib.webpack.isLocal ? 'development' : 'production',
  entry: slsw.lib.entries,
  devtool: slsw.lib.webpack.isLocal ? 'eval-cheap-module-source-map' : 'source-map',
  resolve: {
    extensions: ['.mjs', '.json', '.ts'],
    symlinks: false,
    cacheWithContext: false,
    plugins: [
      new TsconfigPathsPlugin({
        configFile: './tsconfig.paths.json',
      }),
    ],
  },
  output: {
    libraryTarget: 'commonjs',
    path: path.join(__dirname, '.webpack'),
    filename: '[name].js',
  },
  optimization: {
    concatenateModules: false,
  },
  target: 'node',
  externals: [nodeExternals()],
  module: {
    rules: [
      // all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
      {
        test: /\.(tsx?)$/,
        loader: 'ts-loader',
        exclude: [
          [
            path.resolve(__dirname, 'node_modules'),
            path.resolve(__dirname, '.serverless'),
            path.resolve(__dirname, '.webpack'),
          ],
        ],
        options: {
          transpileOnly: true,
          experimentalWatchApi: true,
        },
      },
    ],
  },
  plugins: [],
};
`

I'd appreciate any hint.
